storybook/src/Storybook/BiometricsPopup.qml

```diff
@@ -49,23 +55,12 @@ Dialog {
             horizontalAlignment: Qt.AlignHCenter
             text: "Status Desktop is trying to authenticate you.\n\nTouch ID or enter your password to allow this."
         }
-        StatusButton {
-            Layout.alignment: Qt.AlignHCenter
-            type: StatusBaseButton.Type.Primary
-            focusPolicy: Qt.NoFocus
-            text: "Use password..."
-            onClicked: {
-                root.close()
-                root.obtainingPasswordError("Password required instead of touch ID.", Constants.keychain.errorType.keychain, false)
```

Comment:
This button in MacOS allows to authenticate with OS password instead of a fingerprint. We don't need to cover this specific case here, it doesn't matter for us.

Reply:
Yeah, I will align it.

Reply:
Ah, ok, you already fixed that :cool: